import-service/serverless.yml

```diff
@@ -0,0 +1,54 @@
+service: import-service
+
+frameworkVersion: "3"
+
+provider:
+  name: aws
+  runtime: nodejs18.x
+  stage: dev
+  region: eu-west-1
+
+iamRoleStatements:
+  - Effect: 'Allow'
+    Action:
+      - 's3:ListBucket'
+    Resource: 'arn:aws:s3:::your-home-office-files1'
+  - Effect: 'Allow'
+    Action:
+      - 's3:ListBucket'
+      - 's3:GetObject'
+      - 's3:PutObject'
+    Resource: 'arn:aws:s3:::your-home-office-files1/*'
+
+custom:
+  bundle:
+    sourcemaps: false
+
+functions:
+  importProductsFile:
+    handler: importProductsFile/index.handler
+    events:
+      - httpApi:
+          path: /import
+          method: get
+          cors: true
```

Comment:
for me cors and request are invalid here

Reply:
In my case everything works fine, as I remember. There is a warning, but it still works...